x/emissions/keeper/msg_server_update_params.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/zeta-chain/zetacore/x/emissions/types"
+)
+
+func (k msgServer) UpdateParams(goCtx context.Context, msg *types.MsgUpdateParams) (*types.MsgUpdateParamsResponse, error) {
+	if msg.Authority != k.authority {
+		return nil, errors.Wrapf(govtypes.ErrInvalidSigner, "invalid authority; expected %s, got %s", k.authority, msg.Authority)
+	}
```

Comment:
I am wondering should make this to be 
either authority or emergency group policy ? 

Would be good to get thoughts from @lumtis  as well.

Reply:
yeah , I think the authority approach is correct one , but I was wondering if it might be a good idea to add emergency policy to change this as well